Backend/routes/shows.js

```diff
@@ -1,5 +1,8 @@
 const router = require('express').Router();
 let Show = require('../models/show.model');
+let {checkFunimation} = require('../scraper/checkFunimation');
+let crunchyroll = require('../scraper/crunchyroll')
```

Comment:
a lot of my comments are questions out of curiosity

why do you do {} for some and just the name for others?